corehq/apps/users/models.py

```diff
@@ -2686,6 +2687,8 @@ class Invitation(models.Model):
     profile = models.ForeignKey("custom_data_fields.CustomDataFieldsProfile",
                                 on_delete=models.SET_NULL, null=True)
     custom_user_data = models.JSONField(default=dict)
+    tableau_role = models.CharField(max_length=32, choices=TABLEAU_ROLES, null=True)
+    tableau_group_ids = ArrayField(models.CharField(max_length=36), null=True)
```

Comment:
Would we be able the get the group name from Tableau via the group id? My understanding is that Tableau is the source of truth for available groups

Reply:
Another point on that, I imagine the group name could also change? So if we store the group name, I think there could potentially be a stale data issue where the group name changes between the creation of the invite and the creation of the web user. In that case, the `allowed_groups_for_domain` check would be incorrect.

Reply:
Appreciate your insight and it makes sense. 

I'm imagining manipulation will need to be done anyways to create `TableauGroupTuples` before using `update_tableau_user` so the group names would be retrieved then. I'm leaning towards doing that since storing the group name in the `Invitation` model feels redundant and my preference is to stick closer to the "Tableau is the source of truth" framework. So going to proceed with that